dashboard/src/components/PreservationActionCollapse.vue

```diff
@@ -138,7 +138,7 @@ watch($$(expandCounter), () => show());
             <td>
               <StatusBadge :status="task.status" />
             </td>
-            <td>{{ task.note }}</td>
+            <td class="line-wrap">{{ task.note }}</td>
```

Comment:
@DanielCosme I know I suggested "line-wrap" the other day, but I think "line-break" is actually more accurate.

Reply:
Ok